google/api_core/client_logging.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import json
+import os
+
+from typing import List, Optional
+
+_LOGGING_INITIALIZED = False
+_BASE_LOGGER_NAME = "google"
+
+# Fields to be included in the StructuredLogFormatter.
+#
+# TODO(https://github.com/googleapis/python-api-core/issues/761): Update this list to support additional logging fields.
+_recognized_logging_fields = [
```

Comment:
One thing to note is that users will need to update `google-api-core` to get any additional logging fields in the future.

Reply:
This is true. Any fields that we add in the future will require users to update `google-api-core`.